boards/blackpill-128kib/Kconfig

```diff
@@ -17,4 +17,11 @@ config BOARD_BLACKPILL_128KIB
 
     select HAS_HIGHLEVEL_STDIO
 
+# HACK: This is added due to the make resolution
+# make will select timer backend, probably due to the USBUS
+# and kconfig cannot select if something is already selected like make
+choice ZTIMER_MSEC_BACKEND
+    default ZTIMER_MSEC_BACKEND_TIMER if MODULE_PERIPH_RTC
```

Comment:
Uh shouldn't this be the opposite?
IIUC `periph_rtc` and `periph_rtt` both use the same `RTC` peripheral on stm32f1.
So When the user selects the `periph_rtc` module the `periph_rtt` module used by `ztimer_msec` can't be used.

Reply:
> So when the user selects the `periph_rtc` module the `periph_rtt` module used by `ztimer_msec` can't be used.

If this is the case, we have to implement the delay of 1 ms in `usbdev_fs.c` as busy waiting `nop` loop to avoid that the `periph/usbdev` driver needs the `ztimer_msec` module.

Reply:
There is already the `ztimer_no_periph_rtt` module that can be used to use the usec timer instead for ztimer_msec.

Reply:
Ah sorry I'm confused - this already does exactly that.
It selects the `periph_timer` backend instead of `periph_rtt` if `periph_rtc` is used.